dbt-cta/blocks_incremental/models/1_cta_incremental/_1_cta_incremental__models.yml

```diff
@@ -0,0 +1,4556 @@
+      - name: score
+        description: ''
+      - name: identification
+        description: ''
+      - name: voting_state
+        description: ''
+      - name: id
+        description: ''
+        tests:
+          - unique
+      - name: state_id
+        description: ''
+      - name: longitude
+        description: ''
+      - name: ovr_status
+        description: ''
+      - name: hard_copy_collected
+        description: ''
+      - name: mailing_street_address_two
+        description: ''
+      - name: completed
+        description: ''
+      - name: created_by_user_id
+        description: ''
+      - name: registration_date
+        description: ''
+      - name: secondary_identification
+        description: ''
+      - name: voting_city
+        description: ''
+      - name: phone_number
+        description: ''
+      - name: form_number
+        description: ''
+      - name: party
+        description: ''
+      - name: contacted_voter
+        description: ''
+      - name: district_config
+        description: ''
+      - name: mailing_street_address_one
+        description: ''
+      - name: gender
+        description: ''
+      - name: ethnicity
+        description: ''
+      - name: signature
+        description: ''
+      - name: date_of_birth
+        description: ''
+      - name: latitude
+        description: ''
+      - name: delivery_id
+        description: ''
+      - name: extras
+        description: ''
+      - name: created_at
+        description: ''
+      - name: voting_street_address_two
+        description: ''
+      - name: voting_zipcode
+        description: ''
+      - name: mailing_zipcode
+        description: ''
+      - name: registration_type
+        description: ''
+      - name: redacted_fields
+        description: ''
+      - name: van_id
+        description: ''
+      - name: name_prefix
+        description: ''
+      - name: updated_at
+        description: ''
+      - name: shift_id
+        description: ''
+      - name: first_name
+        description: ''
+      - name: online_registration
+        description: ''
+      - name: por_data
+        description: ''
+      - name: mailing_city
+        description: ''
+      - name: mailing_state
+        description: ''
+      - name: voting_street_address_one
+        description: ''
+      - name: name_suffix
+        description: ''
+      - name: twilio_match_data
+        description: ''
+      - name: last_name
+        description: ''
+      - name: middle_name
+        description: ''
+      - name: us_citizen
+        description: ''
+      - name: precinct
+        description: ''
+      - name: email_address
+        description: ''
+      - name: smarty_streets_match_data
+        description: ''
+      - name: scan_id
+        description: ''
+      - name: attempted
+        description: ''
+      - name: _airbyte_registration_forms_hashid
+        description: ''
+        tests:
+          - unique
```

Comment:
Do we want to have 2 uniqueness tests per table? wondering if just keeping 1 would be good enough especially considering how many tables there are

Reply:
yeah, I did make the deliberate decision to add tests for `id` in addition to `hashid` even though in theory they are duplicative - my inclination is to leave them in for now and remove later if Composer is having trouble executing the dbt in a reasonable amount of time 🫠